StripePayments/StripePaymentsObjcTestUtils/SWHttpTrafficRecorder.m

```diff
@@ -431,6 +436,20 @@ -(void)createMocktailFileWithRequest:(NSURLRequest*)request response:(NSHTTPURLR
     }];
 }
 
+-(NSString *)postBodyRequestStringForRequest:(NSURLRequest *)request {
+    NSString *debugHeader = [request valueForHTTPHeaderField:@"X-Stripe-Mock-Request"];
+    if (debugHeader != nil && [debugHeader length] > 0) {
+        NSString *(^postBodyBlock)(NSURLRequest *request, NSString *postBody) = [SWHttpTrafficRecorder sharedRecorder].postBodyTransformBlock;
+        
+        if(postBodyBlock){
+            debugHeader = postBodyBlock(request, debugHeader);
+        }
```

Comment:
The naming of "postBody" as a parameter in the postBody throws me off here since we are passing in "debugHeader" as a parameter.  Maybe there's a better way to rename postBody?